organization.go

```diff
@@ -24,10 +25,13 @@ type OrganizationChild struct {
 	ID   int64  `json:"id"`
 }
 
-func (c *Client) ListOrganization(minimal bool) ([]*Organization, error) {
+func (c *Client) ListOrganization(filter map[string]string) ([]*Organization, error) {
```

Comment:
I think we should keep the minial option, since it's still possible to use it. And include the filter as the second argument